components/services/services.html

```diff
@@ -0,0 +1,39 @@
+<section class="services">
+  <h1 class="services__header">
+    {{ component.title }}
+  </h1>
+  <div class="boxes">
```

Comment:
What if we tried changing this to a list in yaml, like
boxes:
 - title: foo
   description: bar
 - title: foo2
   description: bar2

And then use a Jinja for loop in the template.
That way we can quickly have an arbitrary number of boxes.